wcfsetup/install/files/style/bbcode/code.scss

```diff
@@ -168,7 +168,7 @@
 	}
 
 	> .toggleButton {
-		background-color: opacify(var(--wcfContentBackground), 0.9);
+		background-color: rgba(var(--wcfContentBackground-rgb) / 0.9);
```

Comment:
I just compared this with scssphp, this change results in a visual change if `$wcfContentBackground` is a color with existing alpha channel:

```
$foo: rgba(255, 0, 27, 0.3);

body {
    color: opacify($foo, 0.5);
}
```
results in:
```
body {
  color: rgba(255, 0, 27, 0.8);
}
```

So any color with more than 10% opacity would no longer have any alpha channel, whereas other colors would have something between 90% and 100% opacity. Now everything has a 90% alpha channel.

Same applies to the other changes here.

Reply:
I’m 99.9% confident that the use of `opacity()` was incorrect before. Both before and after looks the same for the default colors, so it’s very likely that this was an unintentional misuse.

Reply:
> I’m 99.9% confident that the use of opacity() was incorrect before.

(Note: opaciFy, not opacity)

Yeah, I agree with that conclusion, the described behavior of "something between 90 and 100 with the vast majority 100" doesn't appear to be super useful. I just wanted to point out the difference so this can the double-checked.